modules/virt-enlarging-vm-disk.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-vm-disk-resizing.adoc
+
+:_content-type: PROCEDURE
+[id="virt-enlarging-vm-disk_{context}"]
+= Enlarging a virtual machine disk
+
+You can enlarge a virtual machine (VM) disk by resizing the disk's persistent volume claim (PVC).
+
+VM disk enlargement makes extra space available to the virtual machine. However, it is the responsibility of the VM owner to decide how to consume the storage.
+
+Enabling VM disk enlargement notifies the virtual machine about size changes and, if the disk is a `Filesystem` PVC, the matching file expands to the remaining size while reserving some space for file system overhead.
+
+[NOTE]
+====
+You cannot reduce the size of a VM disk.
+====
+
+.Procedure
+
+. Edit a VM disk's PVC by running the following command:
++
+[source,terminal]
+
+----
+$ oc edit pvc <pvc_name>
+----
+
+. Increase the `spec.resource.requests.storage` item to a larger size in the manifest of the VM disk that you want to expand:
++
+[source,yaml]
+
+----
+apiVersion: v1
```

Comment:
As Pan mentioned in their comment, it is good to have the `apiVersion` and `kind` visible, but you don't need the whole manifest. Browsing through our current docs, I see we typically only include a couple of fields in the `metadata`  section. You can keep the `name` field and delete the rest of the `metadata` section. Also delete the `status` section.

Use an ellipsis (...) to show that the file continues after the quoted block.

The final snippet should look something like this:
````
----
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
   name: vm-disk-expand
spec:
  accessModes:
     - ReadWriteMany
  resources:
    requests:
       storage: 3Gi <1>
...
----
````